demo-java/app/src/main/java/com/example/placesdemo/programmatic_predictions/ProgrammaticAutocompleteToolbarActivity.java

```diff
@@ -18,34 +18,57 @@
 import android.os.Handler;
 import android.util.Log;
 import android.view.Menu;
+import android.view.MenuItem;
 import android.widget.ProgressBar;
 import android.widget.SearchView;
 import android.widget.SearchView.OnQueryTextListener;
 import android.widget.ViewAnimator;
+import androidx.annotation.NonNull;
 import androidx.annotation.Nullable;
+import androidx.appcompat.app.AlertDialog;
 import androidx.appcompat.app.AppCompatActivity;
 import androidx.recyclerview.widget.DividerItemDecoration;
 import androidx.recyclerview.widget.LinearLayoutManager;
 import androidx.recyclerview.widget.RecyclerView;
+import com.android.volley.Request.Method;
+import com.android.volley.RequestQueue;
+import com.android.volley.toolbox.JsonObjectRequest;
+import com.android.volley.toolbox.Volley;
 import com.example.placesdemo.MainActivity;
 import com.example.placesdemo.R;
+import com.example.placesdemo.model.GeocodingResult;
 import com.google.android.gms.common.api.ApiException;
+import com.google.android.gms.maps.model.LatLng;
 import com.google.android.libraries.places.api.Places;
 import com.google.android.libraries.places.api.model.AutocompletePrediction;
+import com.google.android.libraries.places.api.model.AutocompleteSessionToken;
+import com.google.android.libraries.places.api.model.LocationBias;
+import com.google.android.libraries.places.api.model.RectangularBounds;
 import com.google.android.libraries.places.api.model.TypeFilter;
 import com.google.android.libraries.places.api.net.FindAutocompletePredictionsRequest;
 import com.google.android.libraries.places.api.net.PlacesClient;
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
 import java.util.List;
+import org.json.JSONArray;
+import org.json.JSONException;
 
 /**
- * An Activity that demonstrates programmatic as-you-type place predictions.
+ * An Activity that demonstrates programmatic as-you-type place predictions. The parameters of the
+ * request are currently hard coded in this Activity, to modify these parameters (e.g. location
+ * bias, place types, etc.), see {@link ProgrammaticAutocompleteToolbarActivity#getPlacePredictions(String)}.
```

Comment:
Either here or at https://github.com/googlemaps/android-places-demos/pull/59/files#diff-a63445244878a277333cf0dfe272666cR154-R155 it would be helpful to link to the docs for full explanation of available parameters:
https://developers.google.com/places/android-sdk/autocomplete#get_place_predictions_programmatically